backend/internal/user/repo/user.go

```diff
@@ -97,6 +123,8 @@ func (r *UserRepo) CreateInviteCode(ctx context.Context, userID string, code str
 	return r.db.InviteCode.Create().
 		SetAdminID(adminID).
 		SetCode(code).
+		SetStatus(consts.InviteCodeStatusPending).
```

Comment:
在CreateInviteCode函数中，设置了邀请码的过期时间为15分钟后。这个时间是否应该作为一个可配置的参数，而不是硬编码在代码中？这样可以提高代码的灵活性和可维护性。

```suggestion
// 可以考虑从配置文件中读取过期时间，例如：
// expireDuration := config.GetInviteCodeExpireDuration()
// SetExpiredAt(time.Now().Add(expireDuration))